content/includes/dockerfile-labs-channel.md

```diff
@@ -1,7 +1,3 @@
----
----
-
-{{% experimental %}}
-The "labs" channel provides early access to Dockerfile features that are not
-yet available in the stable channel.
-{{% /experimental %}}
+> [!NOTE]
```

Comment:
Wonder if we could use the `[!IMPORTANT]` alert for this one: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

(if we support it) but not a blocker